packages/ui/src/elements/Localizer/LocalizerLabel/index.tsx

```diff
@@ -20,7 +20,7 @@ export const LocalizerLabel: React.FC<{
   return (
     <div
       aria-label={ariaLabel || t('general:locale')}
-      className={[baseClass, className].filter(Boolean).join(' ')}
+      className={[baseClass, className, `locale-${locale.code}`].filter(Boolean).join(' ')}
```

Comment:
In the future, useLocale will possibly return null. Can you change the locale part to the following:

```ts
className={[
  baseClass,
  className, 
  locale && `locale-${locale.code}`
].filter(Boolean).join(' ')}
```